library/azure_rm_securitygroup.py

```diff
@@ -95,6 +95,20 @@
                   - Default tags such as C(VirtualNetwork), C(AzureLoadBalancer) and C(Internet) can also be used.
                   - It can accept string type or a list of string type.
                 default: "*"
+            source_application_security_groups:
+                description:
+                    - List of the source application security groups ids.
```

Comment:
no need to have "ids." at the end, as it's only one of available options

Reply:
more than ids. it could resource group + name, or names.